int32 id
int32 age
int32 classification

Point2D velocity
Point2D object_closest
Point2D object_box_size
Point2D[] contour